tidoop-mr-lib/README.md

```diff
@@ -228,14 +229,76 @@ Usage:
          target/tidoop-mr-lib-x.y.z-jar-with-dependencies.jar \
          com.telefonica.iot.tidoop.mrlib.Filter \
          -libjars target/tidoop-mr-lib-x.y.z-jar-with-dependencies.jar \
-         <input_folder> \
-         <output_folder> \
+         <input folder> \
+         <output folder> \
          <regex>
          
 Categories: **transformation**, **generic**, **flat**.
 
 [Top](#top)
 
+###<a name="maponly"></a>MapOnly
```

Comment:
Maybe is more correct in HTML doing:

```
<a name="maponly">MapOnly</a>
```

...although not sure if that breaks Markdown.